cmd/start.go

```diff
@@ -84,3 +92,27 @@ func startHandler(cmd *cobra.Command, _ []string) {
 
 	logs.Info().Msg("shutting down timelock-worker")
 }
+
+func healthHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	fmt.Fprint(w, "OK")
+}
+
+// starts a http server, serving the healthz endpoint.
+func startServer() {
```

Comment:
startServer is too generic, probably here we want something more specific like startHTTPServer.